apps/sms/test/marionette/new_activity_test.js

```diff
@@ -102,7 +102,7 @@ marionette('Messages as "new" activity target', function() {
         return !!window.wrappedJSObject.Compose;
       }), 'XSS should not be performed');
 
-      assert.isTrue(messagesApp.Composer.sendButton.enabled());
+      client.waitFor(() => messagesApp.Composer.sendButton.enabled());
```

Comment:
I've seen a bunch of failures in this file with disabled retry logic - so found several weak places and fixed, it's still possible to see failures with disabled retry logic ~ 1 in 30-40 runs, but it seems related to the Treeherder not sure we can do anything here - something is very slow somewhere.